src/domain/accounts/index.ts

```diff
@@ -19,14 +19,24 @@ export * from "./limits-checker"
 export * from "./account-validator"
 export * from "./primitives"
 
-const UserIdRegex =
+const KratosUserIdRegex =
```

Comment:
we have a big problem with this (in general) we integrated kratos in our domain and that is not good